README.md

```diff
@@ -0,0 +1,109 @@
+
+**Display Date** (see [cheat sheet](https://docs.google.com/document/d/1aqBGdeNeOqB8OQQivgBA7avTL2yngE4QEBapSmH3hrE/edit?ts=5f7752c9%5C#heading=h.ytgiix2d4t2o) for syntax): default "all" if not specified.
+
+**Instance Type** (see [cheat sheet](https://docs.google.com/document/d/1aqBGdeNeOqB8OQQivgBA7avTL2yngE4QEBapSmH3hrE/edit?ts=5f7752c9%5C#heading=h.xmdvclunh3tg) for syntax): default "all" if not specified.
+
+**Server Configuration**: Not available yet.
+
+**User Account Configuration**: Not available yet.
+
+**Mobile Version**: Not available yet.
+
+**Number of posts**: Not available yet.
+
+**Number of Users**: Not available yet.
+
+
+Resources:
+
+[Condition cheat sheet](https://docs.google.com/document/d/1aqBGdeNeOqB8OQQivgBA7avTL2yngE4QEBapSmH3hrE/edit?ts=5f7752c9%5C)
+
+[Notices schema](https://github.com/mattermost/notices/blob/master/notices.schema.json)
+
+## Branches
+
+### `release`
+This branch is for notices live in production. Any notices or changes merged to `notices.json` on the `release` branch are mirrored to https://notices.mattermost.com/. Servers with notices enabled will check this URL daily for new notices or changes that are then propogated to users when they meet the notice conditions.
+
+Branch protection: 
+- Only PM's can merge to the release branch
+- PR's require approval from QA before merge  
+
+### `master`
+This branch is used by QA to test notices. Any notices or changes merged to `notices.json` on the `master` branch are mirrored to https://notices.mattermost.com/pre-release/notices.json. QA can configure test servers to point to this URL and check for new notices or changes that are propogated to users when they meet the notice conditions. QA can also configure the frequency that servers check this URL for updates to speed up testing purposes. 
+
+Branch protection: 
+- PM's and QA can merge or commit directly to the master branch
+- PR's do not require approval to merge
+
+
+## Creating a notice
+
+### Before opening a PR
+
+1. Check how many notices are live in production in `notices.json`: As a general guideline we should aim to have < 5 live notices that could apply to any unique user to avoid spamming. Please reach out to the PM owner for any live notices if we are at the threshold and need to consider removing or replacing a notice. 
+2. Consider your audience: Carefully consider who should see this notice. Do we need to spam all users or should we target System Admins only? 
+3. Consider narrowing your reach: Consider narrowing your target audience further by using additional conditions such as SKU, client, client versions, server versions, server configs 
+
+### Opening a PR 
+Open a PR against the `release` branch to add your notice to the `notices.json`. See this [example PR] XXXXXX will submit a PR for the v5.29 server upgrade
```

Comment:
Shouldn't the PR first be raised against mater for QA to test?

Reply:
The process we've been using so far is for people to submit PR's to the release branch so that two PRs don't have to be made by the submitter to master and release.